src/Pair/toPairs.spec.js

```diff
@@ -57,3 +55,41 @@ test('toPairs', t => {
 
   t.end()
 })
+
+test('toPairs with array', t => {
+  const data = [
+    undefined,
+    null,
+    0,
+    1,
+    '',
+    'string',
+    false,
+    true,
+    {},
+    []
+  ]
+
+  const expected = [
+    undefined,
+    null,
+    0,
+    1,
+    '',
+    'string',
+    false,
+    true,
+    {},
+    []
+  ].map((x, i) => ({
```

Comment:
I'm not sure i am the biggest fan of this formatting. Could we clean it up to be a bit more pointfree, and lets not do it straight from the array